CRM/Contribute/Form/UpdateSubscription.php

```diff
@@ -323,8 +323,7 @@ public function postProcess() {
       }
     }
 
-    $session = CRM_Core_Session::singleton();
-    $userID = $session->get('userID');
+    $userID = CRM_Core_Session::singleton()->get('userID');
```

Comment:
@eileenmcnaughton If updating this line it should be changed to:
```suggestion
    $userID = CRM_Core_Session::getLoggedInContactID();
```

Reply:
@mattwire yeah - I won't add that to this PR thought because - the point of the PR is to add the test. I fixed 2 super obvious things in the class while I was doing it but I don't want to scope creep into cleaning up the form class in this pr

Reply:
I pulled the change out